mirdata/jams_utils.py

```diff
@@ -249,7 +249,7 @@ def jams_converter(
 
 
 def beats_to_jams(beats):
```

Comment:
this file should not be included in this PR

Reply:
Black modifies it.

Reply:
Try running black -S